docs/_pages/community.cn.md

```diff
@@ -72,6 +72,7 @@ Committers are chosen by a majority vote of the Apache Hudi [PMC](https://www.ap
 | <img src="https://avatars.githubusercontent.com/vinothchandar" style="max-width: 100px" alt="vinothchandar" align="middle" /> | [Vinoth Chandar](https://github.com/vinothchandar)           | PMC, Committer | vinoth       |
 | <img src="https://avatars.githubusercontent.com/yanghua" style="max-width: 100px" alt="yanghua" /> | [vinoyang](https://github.com/yanghua)                       | PMC, Committer       | vinoyang     |
 | <img src="https://avatars.githubusercontent.com/lw309637554" style="max-width: 100px" alt="lw309637554" /> | [Wei Li](https://github.com/lw309637554)                       | Committer       | liway     |
+| <img src="https://avatars.githubusercontent.com/zhedoubushishi" style="max-width: 100px" alt="zhedoubushishi" /> | [Wenning Ding](https://github.com/zhedoubushishi)                       | Committer       | wenningd     |
```

Comment:
@zhedoubushishi would you verify your info was correct?

Reply:
Thanks for adding this change. Looks good